whyis/test/api_test_case.py

```diff
@@ -2,4 +2,29 @@
 
 
 class ApiTestCase(TestCase):
-    pass
+
+    def publish_nanopub(self, data, content_type, expected_response=201, login=True):
+        if login:
```

Comment:
login should still be a separate operation. Functions should not change their behavior substantially based on boolean flags.